File: SampleObjectiveC/SampleObjectiveC/ViewController.m
//
//  ViewController.m
//  SampleObjectiveC
//
//  Created by ichise on 2017/07/31.
//

#import "ViewController.h"
@import EntypoSymbol;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    EntypoSymbol *symbol = [[EntypoSymbol alloc] initWithText:[EntypoIcon install] size:30];
    [symbol addAttributeWithAttributeName:NSForegroundColorAttributeName value:[UIColor blackColor]];
    _sampleImageView.image = [symbol imageWithSize:CGSizeMake(30, 30)];
    
    _sampleLabel.font = [EntypoFont fontOfSize:20];
    _sampleLabel.text = [EntypoIcon install];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Sample/Sample/ViewController.swift
//
//  ViewController.swift
//  Sample
//
//  Created by tichise on 2015年5月10日 15/05/10.
//  Copyright (c) 2015年 tichise. All rights reserved.
//

import UIKit
import EntypoSymbol

class ViewController: UIViewController {
    
    @IBOutlet var sampleImageView:UIImageView!
    @IBOutlet var sampleLabelView:UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let symbol:EntypoSymbol = EntypoSymbol(text:EntypoIcon.install as NSString, size:25)
        let iconImage:UIImage = symbol.image(size: CGSize(width: 25, height: 25))
        self.sampleImageView.image = iconImage
        
        sampleLabelView.font = EntypoFont.fontOfSize(40)
        sampleLabelView.text = EntypoIcon.creativeCommons
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}

